src/features/ExtensionCommands.ts

```diff
@@ -364,15 +362,16 @@ export class ExtensionCommandsFeature extends LanguageClientConsumer {
     }
 
     private async openFile(openFileDetails: IOpenFileDetails): Promise<EditorOperationResponse> {
-        const filePath = await this.normalizeFilePath(openFileDetails.filePath);
+        const filePath = await this.resolveFilePathWithCwd(openFileDetails.filePath);
```

Comment:
Renamed this because we're not doing normalization, we're just resolving to the CWD if we have to.